Mathlib/RingTheory/Ideal/Over.lean

```diff
@@ -399,17 +400,64 @@ theorem exists_ideal_over_prime_of_isIntegral (H : Algebra.IsIntegral R S) (P :
     exact congr_arg (comap . Q') (RingHom.ext fun r => rfl)
   · refine' _root_.trans (comap_map_of_surjective _ Quotient.mk_surjective _) (sup_eq_left.2 _)
     simpa [← RingHom.ker_eq_comap_bot] using hIP
+
+lemma exists_ideal_comap_le_prime (P : Ideal R) [P.IsPrime]
```

Comment:
`I` can be implicit here, but as you prefer (similarly below).

Reply:
I think I'll follow the rest of the file and leave it explicit.